Fakebook.Posts/Fakebook.Posts.RestApi/Controllers/PostsController.cs

```diff
@@ -233,5 +239,27 @@ public async Task<IActionResult> GetNewsfeedAsync()
             return NoContent();
         }
 
+        [HttpPost("UploadPicture"), DisableRequestSizeLimit]
+        public async Task<ActionResult> UploadPicture() 
+        {
+            try {
+                if (Request.Form.Files[0] is IFormFile file)
+                {
+                    // generate a random guid from the file name
+                    var newFileName = $"{Request.Form["userId"]}-{Guid.NewGuid()}.{file.FileName.Split('.').Last()}";
+                    // upload image
+                    var result = await _blobService.UploadFileBlobAsync(
+                        "fakebook",
+                        file.OpenReadStream(),
```

Comment:
we may be leaking unmanaged resources here, if this stream is something we need to dispose.

Reply:
Enclosed in a using statement now